keps/sig-storage/3333-reconcile-default-storage-class/README.md

```diff
@@ -425,11 +423,11 @@ Recall that end users cannot usually observe component logs or access metrics.
 
 - [ ] Events
     - Event Reason:
-- [ ] API .status
+- [X] API .spec
     - Condition name:
-    - Other field:
-- [ ] Other (treat as last resort)
-    - Details:
+    - Other field: `pvc.spec.storageClassName` changing from nil to current default StorageClass name after the default is set
+- [X] Other (treat as last resort)
```

Comment:
That's not really true - the metric is an aggregate, I can check if the feature is used, but not if my particular instance of PVC is using it.
Please unmark this one.

Reply:
Done.